tools/wasp-cli/waspcmd/cmd.go

```diff
@@ -60,7 +60,11 @@ func DefaultWaspNodeFallback(node string) string {
 }
 
 func getDefaultWaspNode() string {
-	waspSettings := viper.Sub("wasp").AllSettings()
+	configSubTree := viper.Sub("wasp")
+	if configSubTree == nil {
+		log.Fatal("`wasp` field or config file doesn't exist")
+	}
+	waspSettings := configSubTree.AllSettings()
```

Comment:
```suggestion
    waspSettings := map[string]interface{}{}
	if viper.Sub("wasp") != nil {
		waspSettings = viper.Sub("chains").AllSettings()
	}
```

falling back to `case 0` below.